### DISCUSSION TITLE
How to remove locally install packages

### DISCUSSION THREAD

**Discussion by oraqlle** [vector-of-bool/dds#122] [Q&A] [answered]
Is there a way to remove a specific package version that has been locally stored? 

Say I have package XYZ and I have versions XYZ@0.0.1, XYZ@0.0.2 and XYZ@0.0.3. How would I remove the local cache of say XYZ@0.0.1.

Regards

**Comment by vector-of-bool** [comment, marked as answer]
For alpha.6, the easiest way is to just delete the cached source dist directory.

**Comment by oraqlle** [comment]
Haha, that's what I ended up doing for a temporary solution. Thanks for the response.
